recipes/bob.db.base/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "2.0.6" %}
+
+package:
+  name: bob.db.base
+  version: {{ version }}
+
+source:
+  fn: bob.db.base-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.db.base/bob.db.base-{{ version }}.zip
+  md5: 5a907b00d1336721289cafac6448cb41
+
+build:
+  entry_points:
+  - bob_dbmanage.py = bob.db.base.script.dbmanage:main
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+  - python
+  - setuptools
+  - bob.extension
+  - bob.io.base
+  - sqlalchemy
+  - gcc     # [linux]
+
+  run:
+  - python
+  - bob.extension
+  - bob.io.base
+  - sqlalchemy
+  - libgcc  # [linux]
+
+test:
+  commands:
+  - bob_dbmanage.py --help
+  - nosetests -sv bob.db.base
+
+  imports:
+  - bob
+  - bob.db
+  - bob.db.base
+  - bob.db.base.script
+
+  requires:
+  - nose
+
+about:
+  home: http://github.com/bioidiap/bob.db.base
+  license: Modified BSD License (3-clause)
+  summary: Bob's Basic Database API
+
+extra:
+  recipe-maintainers:
+  - 183amir
```

Comment:
Yeah, this goes beyond style here. It could actually cause problems for various parsers in the automation process.